includes/woopay/class-woopay-session.php

```diff
@@ -350,7 +350,7 @@ private static function get_init_session_request( $order_id = null, $key = null,
 			$customer_id   = WC_Payments::get_customer_service()->create_customer_for_user( $user, $customer_data );
 		}
 
-		if ( 0 !== $user->ID ) {
+		if ( WC_Payments_Features::is_customer_multi_currency_enabled() && ( 0 !== $user->ID ) ) {
```

Comment:
I've always been partial to the parenthesis around conditions like `( 0 !== $user->ID )`, but for consistency we don't use them elsewhere, so we can remove them here.

Reply:
Updated! I guess we'll all just blindly trust operator precedence rules then 😀 . Thanks for reviewing!